src/d3a/models/area/__init__.py

```diff
@@ -190,6 +190,8 @@ def tick(self, is_root_area=False):
             self._cycle_markets()
         self.dispatcher.broadcast_tick()
         self.current_tick += 1
+        for market in self._markets.markets.values():
+            market.update_clock()
```

Comment:
Can you help me understand why `current_tick` has to be re-incremented when its already been incremented two line above?

Reply:
The self.current_tick in this function only increases the number in the area class, but we also want to increase the current_tick member of the market class.